src/udev/udev-builtin-input_id.c

```diff
@@ -235,9 +240,9 @@ static bool test_pointers(sd_device *dev,
                         is_abs_mouse = true;
                 else if (has_touch || is_direct)
                         is_touchscreen = true;
-                else if (has_joystick_axes_or_buttons)
+                else if (num_joystick_buttons || num_joystick_axes)
```

Comment:
We don't do implicit bool conversion for integer types, please use `> 0`.

Reply:
done, thanks

Reply:
sigh, talk about tunnel vision, I changed the one 2 lines south but not this one. Done now, this time for both :)